airbyte-integrations/connectors/source-hubspot/source_hubspot/api.py

```diff
@@ -662,8 +710,12 @@ class CRMObjectIncrementalStream(CRMObjectStream, IncrementalStream):
     limit = 100
     need_chunk = False
 
+    def __init__(self, name: str = None, **kwargs):
+        super().__init__(**kwargs)
+        self._name = name
+
 
-class CampaignStream(Stream):
+class Campaigns(Stream):
```

Comment:
Refactored stream names so its snake case version matches to `schemas/campaigns.json`